Plots/Contours/NCL_conwomap_1.py

```diff
@@ -0,0 +1,63 @@
+"""
+NCL_conwomap_1.py
+=================
+This script illustrates the following concepts:
+   - Drawing a simple line contour plot
+   - Drawing a line contour plot with multiple colors
+   - Increasing the thickness of contour lines
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/conwomap_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/conwomap_1_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+
+import geocat.datafiles as gdf
+from geocat.viz import cmaps as gvcmaps
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/cone.nc"))
+u = ds.u.isel(time=4)
+
+###############################################################################
+# Plot:
+
+#create figure
+plt.figure(figsize=(10,10))
+
+#create axes
+ax = plt.axes(projection=ccrs.PlateCarree())
```

Comment:
Indeed no need for setting `project` as a Cartopy projection here for this example. Tried and saw no difference.

Reply:
See above